### PR TITLE
feat: Add new CR single msg style

### PR DESCRIPTION
**Pull request by gusliedke**
Does #41

### CODE REVIEW

**Comment by gusliedke** [comment]
@KoalaMango done some changes in `kss/index.twig` now we can override default twig components files with our product twig components

File: sass/base/components/single-msg/_single-msg.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Single Message 
 //
-// Markup: _single-msg.twig
+// Markup: single-msg.twig
```
Comment (AndyEPhipps) [inline sass/base/components/single-msg/_single-msg.scss]: I'm guessing this name format change is intentional?

File: sass/base/components/progress-bar/_progress-bar.scss [hunk 2]
```diff
@@ -4,7 +4,7 @@
 //
 // Markup: progress-bar.twig
 //
-// Style guide: Progress bar
+// Style guide: 20.1000
```
Comment (gusliedke) [inline sass/base/components/progress-bar/_progress-bar.scss]: Yes, just want to keep this component at the last of the list

File: sass/base/components/single-msg/_single-msg.scss (hunk 1 above)
Comment (gusliedke) [inline sass/base/components/single-msg/_single-msg.scss]: yes, we don't need to add `_` to twig templates

**Comment by gusliedke** [comment]
@AndyEPhipps can you merge please? Passed design review

**Pull request merged**